src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -33,8 +33,7 @@ end
 Returns the surface height of the `Snow` model.
 """
 function ClimaLand.surface_height(model::SnowModel{FT}, Y, p) where {FT}
-    z_sfc = ClimaCore.Fields.coordinate_field(model.domain.space.surface).z
-    return z_sfc
+    return p.snow.z
```

Comment:
I had to change this to get the snow model to run globally, because if it was defined a point space, there was a `z` coordinate, but if it was a spectral element space, there was not. 

Problems will arise in the surface flux computation, however, if the snow depth is higher than the atmosphere lowest level that we use for the forcing (I think currently a few m). But in principle, this shouldnt happen!

Reply:
The snow level in e.g., Greenland is huge, no? (up to 3km), or is this considered as ice?
(compacted snow becoming ice...) - maybe not a concern for this PR but to keep in mind? What do you think?

Reply:
Great question! I will leave note in the docstring.

I also added an error message so that if we encounter this, we wont just get NaNs, but the simulation will quit on us.

Reply:
I asked Akshay, and he said that the atmos model uses the land ice level as topography.